Code/Framework/AzCore/AzCore/Memory/HphaSchema.cpp

```diff
@@ -140,8 +140,8 @@ namespace AZ {
         class const_iterator;
         class iterator
         {
-            typedef T& reference;
-            typedef T* pointer;
+            using reference = T &;
+            using pointer = T *;
```

Comment:
nitpick: space after the type and all of the using aliases that were added to this file
`using reference = T &;` -> `using reference = T&;`